components/molecules/ContributorProfileInfo/contributor-profile-info.tsx

```diff
@@ -61,20 +65,16 @@ const ContributorProfileInfo = ({
         <div className="flex items-center text-sm gap-3">
           {isConnected && (
             <>
-              {displayLocalTime && (
-                <span className="flex gap-2 items-center">
-                  <Tooltip content="Time zone">
-                    <FiClock />
-                  </Tooltip>
-                  {timezone ? `UTC${getTimezone(timezone)}` : "UTC+1"}
-                </span>
-              )}
-              <span className="flex gap-2 items-center">
-                <Tooltip content="First PR Opened Date">
-                  <AiOutlineGift className="" />
-                </Tooltip>
-                {prFirstOpenedDate ? getFormattedDate(prFirstOpenedDate) : "June 2022"}
-              </span>
+              <div className="flex gap-2 items-center">
+                <div className="flex gap-1 items-center">
+                  <span className="font-semibold text-light-slate-12 text-sm"> {followerCount} </span>
```

Comment:
refactor (blocking): no need to set `text-light-slate-12` as it's the base font color (see https://github.com/open-sauced/app/blob/beta/styles/globals.css#L14)

```suggestion
                  <span className="font-semibold text-sm"> {followerCount} </span>
```

Same for other places in the JSX below.